src/web/mjs/connectors/ManhwaList.mjs

```diff
@@ -0,0 +1,26 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class ManhwaList extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'manhwalist';
+        super.label = 'Manhwa List';
+        this.tags = [ 'manga', 'indonesian'];
+        this.url = 'https://manhwalist.in';
+        this.path = '/manga/?list';
+    }
+    async _getPages(chapter) {
+        const data = await super._getPages(chapter);
+        return data.map(element => this.createConnectorURI(element));
+    }
+    async _handleConnectorURI(payload) {
+        const request = new Request(payload, this.requestOptions);
```

Comment:
If we can download pictures without need of a referer there is no need to write either _handleConnectorURI or __getPages. Did you try without?

Reply:
Yeah, just checked its working fine even without these functions. I have tested in both Aniglitch and ManhwaList.
I'll update the changes